cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc

```diff
@@ -5,68 +5,68 @@
 
 | [source, cypher]
 GRANT ACCESS
-    ON DATABASE[S] {name \| *}
+    ON {DEFAULT DATABASE \| DATABASE[S] {name \| *}}
     TO role[, ...]
-| Allow the specified role or roles to access the database `name` or all databases
+| Allow the specified role or roles to access the default database, the database `name` or all databases
```

Comment:
There are individual files for some of the different privileges, maybe they should be updated too?

```
database/all-management-syntax.asciidoc
database/constraint-management-syntax.asciidoc
database/deny-database-access-syntax.asciidoc
database/deny-database-start-syntax.asciidoc
database/deny-database-stop-syntax.asciidoc
database/grant-database-access-syntax.asciidoc
database/grant-database-start-syntax.asciidoc
database/grant-database-stop-syntax.asciidoc
database/index-management-syntax.asciidoc
database/name-management-syntax.asciidoc
```